MidasCivil_Adapter/CRUD/Read/Loads/AreaDifferentialTemperatureLoad.cs

```diff
@@ -0,0 +1,118 @@
+using BH.Engine.Adapter;
+using BH.oM.Structure.Loads;
+using BH.oM.Structure.Elements;
+using BH.Engine.Reflection;
+
+namespace BH.Adapter.MidasCivil
+{
+    public partial class MidasCivilAdapter
+    {
+        private List<ILoad> ReadAreaDifferentialTemperatureLoads(List<string> ids = null)
+        {
+            List<ILoad> bhomAreaDifferentialTemperatureLoads = new List<ILoad>();
+
+            List<Loadcase> bhomLoadcases = ReadLoadcases();
+            Dictionary<string, Loadcase> loadcaseDictionary = bhomLoadcases.ToDictionary(
+                        x => x.Name);
+
+            string[] loadcaseFolders = Directory.GetDirectories(m_directory + "\\TextFiles");
+
+            foreach (string loadcaseFolder in loadcaseFolders)
+            {
+                string loadcase = Path.GetFileName(loadcaseFolder);
+                List<string> areaDifferentialTemperatureLoadText = GetSectionText(loadcase + "\\THERGRAD");
+                List<string> areaUniformTemperatureLoadText = GetSectionText(loadcase + "\\ELTEMPER");
+                List<string> feMeshComparison = new List<string>();
+                List<string> loadedFEMeshes = new List<string>();
+
+                if (areaDifferentialTemperatureLoadText.Count != 0)
+                {
+                    List<string> areaDifferentialTempatureElements = areaDifferentialTemperatureLoadText.Select(x => x.Split(',')[0]).ToList();
+                    List<string> areaUniformTempatureElements = areaUniformTemperatureLoadText.Select(x => x.Split(',')[0]).ToList();
+                    for (int i = 0; i < areaDifferentialTempatureElements.Count; i++)
+                    {
+                        List<string> differentialDelimitted = areaDifferentialTemperatureLoadText[i].Split(',').ToList();
+                        loadedFEMeshes.Add(areaDifferentialTempatureElements[i].Trim());
+                        differentialDelimitted.RemoveAt(0);
+                        if (areaUniformTempatureElements.Count != 0)
+                        {
+                            if (areaDifferentialTempatureElements[i] == areaUniformTempatureElements[i])
+                            {
+                                List<string> globalDelimitted = areaUniformTemperatureLoadText[i].Split(',').ToList();
+                                differentialDelimitted.Add(globalDelimitted[1].Trim());
+                            }
+                            else
+                            {
+                                Compute.RecordWarning("No Area Uniform Temperature Load is integrated as part of the Area Differential Temperature Load at Element number " + areaDifferentialTempatureElements[i].Trim().ToString() + " due to the two loads are assigned to differnt element");
+                                Compute.RecordWarning("Area Differential Temperature load will be applied at the centroid of the cross section at Element number " + areaDifferentialTempatureElements[i].Trim().ToString());
```

Comment:
This should be a single warning, make use of the special characters to format it correctly and add new lines so they are readable.